common/app/common/NewNavigation.scala

```diff
@@ -223,41 +216,34 @@ object NewNavigation {
     val name = ""
 
     val uk = NavLinkLists(List(
-      NavLink("apps", "/global/ng-interactive/2014/may/29/-sp-the-guardian-app-for-ios-and-android"),
-      NavLink("jobs", "https://jobs.theguardian.com/?INTCMP=NGW_TOPNAV_UK_GU_JOBS"),
-      NavLink("dating", "https://soulmates.theguardian.com/?INTCMP=NGW_TOPNAV_UK_GU_SOULMATES"),
+      apps.copy(apps.url + "?apps_uk_web_newheader"),
```

Comment:
Haha! This is a result of me trying to multitask and doing it unsuccessfully. Fixed now.